AUTHORS

```diff
@@ -1 +1,2 @@
 Richard Hughes <richard@hughsie.com>
+NVIDIA CORPORATION
```

Comment:
Headers okay, this is a but much. Authors is for substantial amounts of work imho.

Reply:
That's fair. I wasn't sure whether this should be updated or not.